packages/react/src/lib/accessibility/FocusZone/AutoFocusZone.tsx

```diff
@@ -20,51 +12,34 @@ import * as customPropTypes from '../../customPropTypes'
 
 /** AutoFocusZone is used to focus the first element inside it's children */
 export class AutoFocusZone extends React.Component<AutoFocusZoneProps, {}> {
-  private static _focusStack: AutoFocusZone[] = []
   private _root: { current: HTMLElement | null } = { current: null }
   private _previouslyFocusedElementOutsideTrapZone: HTMLElement
```

Comment:
`_previouslyFocusedElementOutsideTrapZone ` it is not set anywhere. In FTZ, it is set usually to document.active element, so on `componentWillUnmount` we can focus it, basically does same as in Popup's action - `closeAndFocusTrigger`. I don't think we need it here.

Reply:
I removed it, but I am a bit concern, as this may introduce some regressions. Previously we had the following code:
```
  public componentDidMount(): void {
    const { disableFirstFocus = false } = this.props

    if (
      !this._root.current.contains(this._previouslyFocusedElementOutsideTrapZone) &&
      !disableFirstFocus
    ) {
      this._findElementAndFocusAsync()
    }
  }
```
Now we don't have the fist condition here...

Reply:
yep, if look in FTZ `_previouslyFocusedElementOutsideTrapZone` is used in many places and is calculated there. Here it doesn't make sense. But we need somehow to respect this condition when `FocusTrapZone` will use `AutoFocusZone`